feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -0,0 +1,107 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { Configurator } from '../../core/model/configurator.model';
+
+export interface ConfiguratorPriceComponentOptions {
+  quantity?: number;
+  price?: Configurator.PriceDetails;
+  priceTotal?: Configurator.PriceDetails;
+  isLightedUp?: boolean;
+}
+
+@Component({
+  selector: 'cx-configurator-price',
+  templateUrl: './configurator-price.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfiguratorPriceComponent {
+  @Input() formula: ConfiguratorPriceComponentOptions;
+
+  /**
+   * Retrieves price.
+   *
+   * @return {string} - value price formula
+   */
+  get price(): string {
```

Comment:
can we order the functions? make 2 get (`get price()`, and `get priceTotal()`) closer, and make `displayPriceOnly` and `displayQuantityAndPrice` closer?

Reply:
picked

Reply:
done